source/includes/connect/stable-api.php

```diff
@@ -0,0 +1,13 @@
+// start-specify-v1
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = ['serverApi' => new MongoDB\Driver\ServerApi('1')];
+$client = new MongoDB\Client($uri, [], $driverOptions);
```

Comment:
```suggestion

$client = new MongoDB\Client($uri, [], $driverOptions);
```

Consider for readability.